gravitee-apim-cypress/cypress/integration/nrt/apis/application-tests.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { ADMIN_USER, API_PUBLISHER_USER, LOW_PERMISSION_USER } from '@fakers/users/users';
+
+context('Application tests', () => {
+  describe('Create an Application', function () {
+    after(function () {
```

Comment:
should be after each

Reply:
Well, I was thinking that I'll only have one happy path and several failing tests which would mean that there's only one application to delete for the whole set of 'Create an Application' tests.

Reply:
Alright, I changed it.